# float64 time_elapsed
float64 translation_norm
float64 rotation_norm
float64 num_surf_from_scan
float64 num_corner_from_scan
